scripts/tests/test_validate_docstrings.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import pandas as pd
 
-import validate_docstrings
+from scripts import validate_docstrings
```

Comment:
I don't see how this is necessary - can you revert this (or let me know what I am missing)

Reply:
no, my bad i wans't working for me when debugging, i reverted this.